source/index.js

```diff
@@ -2,23 +2,25 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-import Junkership from "./Junkership.js"
-import Reference from "./Reference.js"
-import Score from "./Score.js"
-import GameContainer from "./GameContainer.js"
-import PowerUp from "./PowerUp.js"
-import Sound from "./Sound.js"
-import Trashbot from "./Trashbot.js"
+import Junkership from "scripts/sprites/Junkership.js"
+import Reference from "scripts/data/Reference.js"
+import Score from "scripts/ui/Score.js"
+import GameContainer from "scripts/sprites/GameContainer.js"
+import PowerUp from "scripts/PowerUp.js"
+import Sound from "scripts/layers/Sound.js"
+import Trashbot from "scripts/sprites/Trashbot.js"
 
 var renderer = Pixi.autoDetectRenderer(Reference.GAME_WIDTH, Reference.GAME_HEIGHT)
 renderer.backgroundColor = 0x222222
 renderer.roundPixels = true
 
 var mount = document.getElementById("mount")
 mount.insertBefore(renderer.view, mount.firstChild)
+
+// Oof.. a global variable??
 window.game = new GameContainer()
```

Comment:
One of the cool things about Pixi is that everything is doubly-linked; when you `.add()` a sprite to a container, `.parent` in sprite now refers to the container, so we could probably eliminate some of the references to the global variable that way.

But that said, I don't think that the sprites are the only parts of our codebase that uses the global variable, so that approach won't work for everything. :\